nautobot/extras/api/nested_serializers.py

```diff
@@ -222,7 +230,19 @@ class Meta:
         fields = ["id", "url", "name", "slug"]
 
 
-class NestedScheduledJobSerializer(BaseModelSerializer):
+class NestedScheduledJobSerializer(WritableNestedSerializer):
+    url = serializers.HyperlinkedIdentityField(view_name="extras-api:scheduledjob-detail")
+    name = serializers.CharField(max_length=255, required=False)
+    start_time = serializers.DateTimeField(format=None, required=False)
+
+    class Meta:
+        model = models.ScheduledJob
+        fields = ["url", "name", "start_time", "interval", "crontab"]
+
+
+class NestedScheduledJobCreationSerializer(BaseModelSerializer):
```

Comment:
Just calling attention to this addition in my last commit. When we specify schedule information on a JobInputSerializer, this is *not* a standard use of a nested serializer, as it's defining a new schedule for the job, *not* referencing an existing ScheduledJob instance. `WritableNestedSerializer` expects that any parameters you provide it are meant to be used to locate an existing instance and will throw a ValidationError if it doesn't find one, so for the JobInputSerializer case I had to create this separate class which is *not* a WritableNestedSerializer.

Reply:
Will do.